pkg/network/ebpf/c/runtime/http.c

```diff
@@ -794,17 +806,6 @@ int uprobe__crypto_tls_Conn_Read__return(struct pt_regs *ctx) {
     // For now for success validation we chose to check only the amount of bytes read
```

Comment:
Is this comment still relevant? What are we checking to be greater than zero?

Reply:
yes, moved that above the relevant code